agent/handlers/agentapi/v1/taskprotection/types/types.go

```diff
@@ -44,14 +43,8 @@ func (taskProtection *taskProtection) MarshalJSON() ([]byte, error) {
 
 // Creates a taskProtection value after validating the arguments
 func NewTaskProtection(protectionEnabled bool, expiresInMinutes *int) (*taskProtection, error) {
-	if protectionEnabled {
-		if expiresInMinutes == nil {
-			return nil, errors.New("expiration duration is required for enabled task protection")
-		}
-
-		if *expiresInMinutes <= 0 {
-			return nil, fmt.Errorf("expiration duration must be greater than zero minutes for enabled task protection")
-		}
+	if protectionEnabled && expiresInMinutes != nil && *expiresInMinutes <= 0 {
```

Comment:
Is there going to be a default value if unspecified? Or will there by no expiry in that case?

Reply:
Default value will be 8 hrs and this specific case will be handled by the Scheduler

Reply:
In that case will Control Plane issue a task stop? How would ECS Agent know whether its local authority on the task protection should be overridden (because ECS Agent still thinks the task as protected)?

Reply:
There will be no local authority in phase 1, agent will not keep state. If there is phase 2, we can change this behavior.

Reply:
Alright sounds good.